src/promptflow/promptflow/_cli/_pf/_connection.py

```diff
@@ -16,7 +16,6 @@
 from promptflow._sdk.entities._connection import _Connection
```

Comment:
let's get this merged soon, so we can have some e2e test before our next release cut

Reply:
ok